entries/d/dynadot.com.json

```
@@ -3,9 +3,10 @@
     "domain": "dynadot.com",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "u2f"
     ],
-    "documentation": "https://dynadot.com/domain/security.html",
+    "documentation": "https://www.dynadot.com/domain/security",
```

```suggestion
    "documentation": "https://www.dynadot.com/community/help/question/2-step-verification",
```
This doc page might be more useful?